run_operations.py

```diff
@@ -119,168 +128,181 @@ def command_line_parse(args):
 	logger.info("\t\t--startpysar       : %s\n", inps.startpysar)
 	logger.info("\t\t--startinsarmaps   : %s\n", inps.startinsarmaps)
 	logger.info("\t\t--testsheet	    : %s\n", inps.test_sheet)
-	
 
-	
-###################### Auxiliary Functions #####################
 
+###################### Auxiliary Functions #####################
 
-"""
-    Reads template file for the current dataset and parses out the 'ssaraopt' option before creating command options line options 
-    array for ssara_federated_query.py
-    Parameters: none
-    Returns:    ssara_options: [str], array of command line options to run ssara_federated_query with
-"""
 def create_ssara_options():
-	
-	with open('/nethome/'+user+'/insarlab/OPERATIONS/TEMPLATES/'+dataset+'.template', 'r') as template_file:
+	""" Reads template file for the current dataset and parses out the 'ssaraopt' option before creating command options
+		line options array for ssara_federated_query.py
+
+		Parameters: none
+		Returns:    ssara_options: [str], array of command line options to run ssara_federated_query with
+	"""
+
+	with open('/nethome/' + user + '/insarlab/OPERATIONS/TEMPLATES/' + dataset + '.template', 'r') as template_file:
```

Comment:
Seems like an overly complicated and system-dependent method of finding the HOME path. See https://stackoverflow.com/a/4028943/8878070. Let's try to refactor out `user` in the process

Reply:
Yes. Falk has a better way using os.getenv(). Need to update that piece